File: ANDROID/SMA_v1/lib/models/cateSteelDefectModel.dart
class CateSteelDefectModel {
  final int? steelDefectId;
  final String? defectName;
  final String? defectType;
  final int? parentId;
  final String? material;

  CateSteelDefectModel(
      {this.steelDefectId,
      this.defectName,
        this.defectType,
      this.parentId,
      this.material});

  Map<String, dynamic> toJson() {
    return {
      'steelDefectID': steelDefectId,
      'defectName': defectName,
      'defectType': defectType,
      'parentID': parentId,
      'material': material,
    };
  }

  factory CateSteelDefectModel.fromJson(Map<String, dynamic> json) {
    return CateSteelDefectModel(
      steelDefectId: json['steelDefectID'],
      defectName: json['defectName'],
      defectType: json['defectType'],
      parentId: json['parentID'],
      material: json['material'],
    );
  }
  @override
  String toString() {
    // TODO: implement toString
    return 'SteelDefectID: $steelDefectId, DefectName: $defectName,DefectType: $defectType, ParentID: $parentId,Material: $material \n';
  }
}

File: API/Model/Models/CateSteelDefectModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Model.Models
{
    public class CateSteelDefectModel
    {
        public int SteelDefectID { get; set; }
        public string? DefectName { get; set; }
        public string? DefectType { get; set; }
        public int? ParentID { get; set; }
        public string? Material { get; set; }
    }
}
